packages/vbridge/src/hooks/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './bridge/use-bridge';
```

Comment:
can we change the file name so that it is consistent. Please use camelCasing here.